px4_ros2_cpp/include/px4_ros2/utils/geometry.hpp

```diff
@@ -142,34 +142,24 @@ Eigen::Quaternion<Type> eulerRpyToQuaternion(const Type roll, const Type pitch,
  * @brief Convert quaternion to roll angle in extrinsic RPY order (intrinsic YPR)
  *
  * @param q The input quaternion
- * @return Roll angle corresponding to the given quaternion in range [-pi, pi]
+ * @return Roll angle corresponding to the given quaternion in range [-pi/2, pi/2]
```

Comment:
```suggestion
 * @return Roll angle corresponding to the given quaternion in range [-pi, pi]
```
This should be unchanged, `quaternionToEulerRpy` returns euler angles in range R, P, Y = [-pi, pi], [-pi/2, pi/2], [-pi, pi].